docs/rst/fastdds_qos_profiles_manager_lib/api_reference.rst

```diff
@@ -1,12 +1,6 @@
 .. _fastdds_qos_profiles_manager_lib_api_reference:
 
-Fast DDS QoS Profiles Manager Library API Reference
-===================================================
+Introduction
+============
 
-.. toctree::
-
-    /rst/fastdds_qos_profiles_manager_lib/participant
-    /rst/fastdds_qos_profiles_manager_lib/datareader
-    /rst/fastdds_qos_profiles_manager_lib/datawriter
-    /rst/fastdds_qos_profiles_manager_lib/transport_descriptor
-    /rst/fastdds_qos_profiles_manager_lib/exception
```

Comment:
I would keep this even if it is actually redundant.

Reply:
You cannot keep both because Sphinx issues a warning :smiling_face_with_tear: You can try and see for yourself